### PR TITLE
Chore: Bump grafana dependencies

### PR DESCRIPTION
**Pull request by mdvictor**
This PR bumps scenes grafana dependencies to latest version v11.3

### CODE REVIEW

File: packages/scenes-react/package.json [hunk 1]
```diff
@@ -37,16 +37,16 @@
...
   },
   "peerDependencies": {
-    "@grafana/data": "^11.0.0",
-    "@grafana/runtime": "^11.0.0",
-    "@grafana/schema": "^11.0.0",
-    "@grafana/ui": "^11.0.0",
+    "@grafana/data": "^11.3.0",
+    "@grafana/runtime": "^11.3.0",
+    "@grafana/schema": "^11.3.0",
+    "@grafana/ui": "^11.3.0",
```
Comment (jackw) [inline packages/scenes-react/package.json]: nit: I've not looked at the code changes but unless scenes-react now requires ^11.3.0 due to api changes in those packages there shouldn't be a need to change this.

File: packages/scenes-react/package.json [hunk 2]
```diff
@@ -37,16 +37,16 @@
     "url": "https://github.com/grafana/scenes/issues"
   },
   "dependencies": {
-    "@grafana/e2e-selectors": "^11.0.0",
+    "@grafana/e2e-selectors": "^11.3.0",
```
Comment (jackw) [inline packages/scenes-react/package.json:40]: I think we should move this to a peerDep. Maybe we get [this PR](https://github.com/grafana/scenes/pull/940) reviewed and merged?

File: packages/scenes-react/package.json (hunk 1 above)
Comment (mdvictor) [reply packages/scenes-react/package.json]: Yeah, this makes sense, there shouldn't be a need to change this. Thank you!

File: packages/scenes-react/package.json (hunk 2 above)
Comment (mdvictor) [reply packages/scenes-react/package.json:40]: Sounds good! Seems like it already got approved!

Comment (jackw) [reply packages/scenes-react/package.json]: Tis merged now. I apologise for any conflicts. 😬

**Review by torkelo — APPROVED** [review]

**Pull request merged**